bigquery_etl/shredder/delete.py

```diff
@@ -221,7 +221,7 @@ def create_job(client):
             {"DELETE" if use_dml else "SELECT * FROM"}
               `{sql_table_id(target)}`
             WHERE
-              {"" if use_dml else "NOT "}({field_condition})
+             ({field_condition}){"" if use_dml else " IS NOT TRUE"}
```

Comment:
Indentation slightly off.
```suggestion
              ({field_condition}){"" if use_dml else " IS NOT TRUE"}
```